Cython/Compiler/Builtin.py

```diff
@@ -592,6 +594,7 @@ def get_known_standard_library_module_scope(module_name):
         var_entry.is_pyglobal = True
         var_entry.scope = mod
         entry.as_variable = var_entry
+        entry.known_standard_library_import = "%s.%s" % (module_name, name)
```

Comment:
For the typing module, `name = EncodedString(name)` for whichever attribute of the module it is. For dataclasses we don't have that.

Reply:
```suggestion
        entry.known_standard_library_import = "%s.InitVar" % module_name
```